apps/glusterfs/app_config.go

```diff
@@ -23,13 +24,17 @@ type GlusterFSConfig struct {
 	Executor   string              `json:"executor"`
 	Allocator  string              `json:"allocator"`
 	SshConfig  sshexec.SshConfig   `json:"sshexec"`
+	CaiConfig  caiexec.CaiConfig   `json:"caiexec"`
 	KubeConfig kubeexec.KubeConfig `json:"kubeexec"`
 	Loglevel   string              `json:"loglevel"`
 
 	// advanced settings
 	BrickMaxSize int `json:"brick_max_size_gb"`
 	BrickMinSize int `json:"brick_min_size_gb"`
 	BrickMaxNum  int `json:"max_bricks_per_volume"`
+
+	// volume expand method
```

Comment:
fixed 100be448505e88850cf69e7dfc0bce2fe6d49856 

erase the config argument, expand by quota if executor is `raw` by now, otherwise by brick like the old way.